server/data_access_layer/user.js

```diff
@@ -27,7 +38,22 @@ async function updateAnsweredQuestions(user, answeredQuestions) {
   return user
 }
 
+async function registerUser(fullName, email, title, company) {
+  const newUser = [{
+    fields: {
+      'Email': email,
+      'Company': [ company ],
+      'Current Title': title,
+      'Full Name': fullName
+    }
+  }]

```

Comment:
you are actually creating an array here with only one user. you can simply do
```js
const newUser = {
    'Email': email,
    'Company': [ company ],
    'Current Title': title,
    'Full Name': fullName
  }
```
This should also get rid of an issue w/ your tests